detection-rules/link_coinbase_phishing_suspicious_link.yml

```diff
@@ -0,0 +1,17 @@
+name: "Brand impersonation: Coinbase with suspicious link"
+description: "This rule detects Coinbase phishing emails with suspicious links"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and sender.email.domain.root_domain != "coinbase.com"
+  and length(body.links) ==1 
+  and (
+      regex.icontains(coalesce(sender.display_name, subject.subject),
+        '\b[CⒸⓒ𝑪𝒞𝓬𝔠𝕔𝕥𝖈𝖔𝖙𝗌𝗰𝘊𝘤𝙤𝚌𝝰][oₒⓞ𝑜𝒐𝓸𝕠𝖔𝗼𝘐𝙤𝚘𝞸][i𝐼𝑖𝒊𝓲𝕚𝖎𝗜𝘪𝙄𝚒][nⓃ𝑛𝒏𝓷𝔫𝕟𝖓𝙣𝚗]\s?[BⒷⓑ𝑩𝒃𝓫𝔟𝕓𝕥𝗕][aaₐ𝖆𝗮𝘢𝙖𝚊𝛂𝜶𝞪][s𝑆𝒔𝓢𝔰𝕊𝕊𝖘𝖘𝗦𝘴𝙎𝚂][e𝑒𝒆𝓔𝔢𝕖𝕖𝖊𝖊𝗲𝙀𝚎𝝴𝞮]\b')
```

Comment:
will this match ascii "coinbase"? if so, i'd worry a little bit about this FPing on marketing emails with "coinbase" in the subject which can have free file hosting links like firebase etc for hosting images. if you agree, we could just re-add LA to this and accept that it won't run in HIP

Reply:
for sure, this actually worked better than LA in testing, so I'll add FTS.